corehq/apps/app_manager/views/cli.py

```diff
@@ -31,18 +41,61 @@ def app_to_json(app):
 
 @json_error
 def direct_ccz(request, domain):
-    if 'app_id' in request.GET:
-        app = get_app(domain, request.GET['app_id'])
-        app.set_media_versions(None)
-        download = DownloadBase()
-        build_application_zip(
-            include_multimedia_files=False,
-            include_index_files=True,
-            app=app,
-            download_id=download.download_id,
-            compress_zip=True,
-            filename='{}.ccz'.format(slugify(app.name)),
-        )
-        return DownloadBase.get(download.download_id).toHttpResponse()
-    msg = "You must specify `app_id` in your GET parameters"
-    return json_response({'status': 'error', 'message': msg}, status_code=400)
+    """
+    You must specify an app_id, and you may specify either 'version' or 'latest'
+    latest can be one of:
+        release: Latest starred version
+        build: Latest version regardless of star
+        save: Latest saved version of the application (even without a build)
+    If 'version' and 'latest' aren't specified it will default to latest save
+    """
```

Comment:
@amstone326 This is the closest thing to documentation I've written thus far (the error messages should be pretty explanatory though).  Do you know of a related place on confluence to stick this?